paasta_tools/setup_kubernetes_cr.py

```diff
@@ -167,16 +168,16 @@ def setup_all_custom_resources(
             continue
 
         # by convention, entries where key begins with _ are used as templates
-        raw_config_dicts = load_all_configs(
+        config_dicts = load_all_configs(
```

Comment:
we'll need to also add flink to the `paasta_tools::public_config::auto_config_instance_types_enabled` list in puppet

Reply:
Yes I would do that 

[DraftPR](https://github.yelpcorp.com/sysgit/puppet/pull/7734)